src/components/navigation/NavItems.js

```diff
@@ -2,7 +2,7 @@
  * @flow
  */
 
-import React, { useState } from 'react';
+import React, { Fragment, useState } from 'react';
```

Comment:
Are you using `Fragment` anywhere?

Reply:
nope, i removed it—i was using it before but the idea didn't work.